socorro/scripts/add_crashid_to_queue.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
```

Comment:
These scripts get called from `socorro-cmd` and we don't expect anyone to use them directly, so we can nix all this runtime stuff.